src/utils/config.ts

```diff
@@ -2,7 +2,7 @@ import {Alert, Linking} from 'react-native';
 import {check, request, RESULTS} from 'react-native-permissions';
 import SInfo from 'react-native-sensitive-info';
 
-const ROOT_HTTP = 'https://testnet-api.casperdash.io';
+const ROOT_HTTP = __DEV__ ? 'https://testnet-api.casperdash.io' : 'https://api.casperdash.io';
```

Comment:
please add and export NETWORK_NAME
`const NETWORK_NAME = __DEV__ ? 'casper-test' : 'casper'`
it could be used in `/casperdash-mobile-wallet/src/utils/constants/key.js`